### PR TITLE
Add systemd unit config checker

### PR DESCRIPTION
**Pull request by holomorph**
adds a checker and entry in languages.rst for a systemd unit config checker using systemd-analyze. see #1135

### CODE REVIEW

**Comment by manuel-uberti** [comment]
Thanks for the PR. :-)

**Comment by lunaryorn** [comment]
@holomorph Please sign the CLA.  We're not able to accept your contribution otherwise.

**Comment by holomorph** [comment]
boom

**Comment by holomorph** [comment]
@lunaryorn done

**Comment by lunaryorn** [comment]
LGTM

File: doc/languages.rst [hunk 1]
```diff
@@ -1083,6 +1083,13 @@ to view the docstring of the syntax checker.  Likewise, you may use
 
       Check SQL syntax with `Sqlint <https://github.com/purcell/sqlint>`_.
 
+.. supported-language:: systemd Unit Configuration
+
+   .. syntax-checker:: systemd-analyze
+
+      Check systemd unit configuration file syntax with
+      `systemd-analyze <https://www.freedesktop.org/software/systemd/man/systemd-analyze.html>`_.
```
Comment (lunaryorn) [inline doc/languages.rst]: Could you turn this into a "named" link?  e.g.

```
`systemd-analyze`_

.. _systemd-analyze: https://…
```
It's easier to read in plain text form that way.

Don't forget to refill the paragraph afterwards with `M-q`.

**Comment by holomorph** [comment]
@lunaryorn done

**Comment by lunaryorn** [comment]
@holomorph Thanks a lot, merged!

**Comment by cpitclaudel** [comment]
@holomorph Congrats on this contribution!

**Comment by manuel-uberti** [comment]
@holomorph great stuff, thanks!

**Pull request merged**